url78eec0e0b6_Orikuro_vic_tgz-pJ8-net_nexon_vindictus_itemcomparer_inport_StaticImportJ8/src/net/nexon/vindictus/itemcomparer/inport/StaticImport.java

```diff
@@ -0,0 +1,2125 @@
+				134, 28, 14);
+		Helm hh = new Helm(iset, "Rabbit Hat", 60, ItemTyp.CLOTH, 299, 0, 129,
+				29, 9);
+
+		shoes.add(ss);
+		gloves.add(gg);
+		pants.add(pp);
+		armors.add(aa);
+		helms.add(hh);
+
+		sets.add(setname + " 4*");
+		Shoes sss = new Shoes(iset, name, 60, ItemTyp.CLOTH, def, str, vint,
+				agi, crit_res);
+		Pants ppp = new Pants(iset, "Mini Dress", 60, ItemTyp.CLOTH, 318, 0,
+				140, 28, 13);
+		Gloves ggg = new Gloves(iset, "Belt Gloves", 60, ItemTyp.CLOTH, 249, 0,
+				125, 29, 9);
+		Armor aaa = new Armor(iset, "Chain Boleru", 60, ItemTyp.CLOTH, 360, 0,
+				134, 28, 14);
+		Helm hhh = new Helm(iset, "Rabbit Hat", 60, ItemTyp.CLOTH, 299, 0, 129,
+				29, 9);
+
+		sss.setStars(4);
+		ppp.setStars(4);
+		ggg.setStars(4);
+		aaa.setStars(4);
+		hhh.setStars(4);
+
+		shoes.add(sss);
+		gloves.add(ggg);
+		pants.add(ppp);
+		armors.add(aaa);
+		helms.add(hhh);
+
+	}
+
+	static void generateMhara() {
+		String setname = "Mhara";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		int sdef = 195;
+		int sstr = 0;
+		int sint = 0;
+		int sagi = 0;
+		int satk = 0;
+		boni.add(new ItemSetBonus(sdef, sstr, sint, sagi, satk));
+		boni.add(new ItemSetBonus(230, 0, 0));
+		boni.add(new ItemSetBonus(270, 25, 110));
+		boni.add(new ItemSetBonus(330, 45, 160));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(magic);
+
+		Gloves g = new Gloves(iset, "Armguard", 70, ItemTyp.LIGHT, 403, 9, 114,
+				29, 10);
+		String name = "Boots";
+		int def = 405;
+		int str = 9;
+		int vint = 114;
+		int agi = 30;
+		int crit_res = 10;
+		Shoes s = new Shoes(iset, name, 70, ItemTyp.LIGHT, def, str, vint, agi,
+				crit_res);
+		Pants p = new Pants(iset, "Chaps", 70, ItemTyp.LIGHT, 485, 11, 151, 33,
+				12);
+		Helm h = new Helm(iset, "Fur Cap", 70, ItemTyp.LIGHT, 459, 10, 130, 30,
+				11);
+		Armor a = new Armor(iset, "Scale Coat", 70, ItemTyp.LIGHT, 536, 12,
+				151, 33, 14);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
+
+	}
+
+	static void generatePearlClassic() {
+		String setname = "Classic Black Pearl";
+		sets.add(setname);
+		List<ItemSetBonus> boni = new ArrayList<ItemSetBonus>();
+		boni.add(new ItemSetBonus(225, 0, 80));
+		boni.add(new ItemSetBonus(260, 0, 110));
+		boni.add(new ItemSetBonus(300, 0, 130));
+		boni.add(new ItemSetBonus(360, 0, 180));
+
+		ItemSet iset = new ItemSet(setname, boni);
+		iset.setUsefulChars(magic);
+
+		Shoes s = new Shoes(iset, "Boots", 80, ItemTyp.HEAVY, 737, 29, 125, 45,
+				12);
+		Gloves g = new Gloves(iset, "Gauntlets", 80, ItemTyp.LIGHT, 723, 30,
+				125, 45, 12);
+		Pants p = new Pants(iset, "Greaves", 80, ItemTyp.LIGHT, 798, 35, 151,
+				49, 15);
+		Helm h = new Helm(iset, "Helmet", 80, ItemTyp.LIGHT, 772, 34, 132, 46,
+				14);
+		Armor a = new Armor(iset, "Mail", 80, ItemTyp.LIGHT, 845, 36, 159, 50,
+				16);
+
+		shoes.add(s);
+		gloves.add(g);
+		pants.add(p);
+		armors.add(a);
+		helms.add(h);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Similar code fragments were detected in the same file at the following lines: 194:220, 225:251, 256:283, 288:313, 349:375, 412:437, 442:464, 469:495, 500:525, 530:556, 561:586, 591:616, 621:646, 651:676, 988:1011, 1043:1068, 1072:1097, 1318:1342, 1346:1369, 1373:1398, 1431:1456, 1460:1485, 1489:1514, 1549:1573, 1577:1601, 1605:1630, 1668:1691.  
Refactoring can help improve code maintainability. Consider reducing duplicate code by extracting it into a separate method. You can then replace duplicated code with calls to this new method.